data/campaigns/Northern_Rebirth/scenarios/12a_Get_the_Gold.cfg

```diff
@@ -23,74 +23,82 @@
+        {STORY_PART_ERYSSA (_ "Hidel, I am sorry. I have sent you to your death.")}
 
         {STORY_PART_HIDEL (_ "... Don’t tax yourself... Eryssa... I have died... a warrior’s death... You should be proud...")}
 
         {STORY_PART_HIDEL (_ "... But... there is... one thing... we... could do. We... managed to... recover the gold... Sisal... took it and... retreated south...")}
 
         {STORY_PART_HIDEL (_ "I held out as... long as... I could... but they... defeated us... Rakshas went... east and the... rest... went... after... Sisal...")}
 
-        {STORY_PART_HIDEL (_ "Now... uphold the honor... of the Northern... Elves... Recover... the gold... slay this monster... and bring peace and... prosperity... back to the... Northlands... Tallin?")}
+        {STORY_PART_HIDEL (_ "Now... uphold the honor... of the Northern... Elves... Recover... the gold... slay this monster... and bring peace... to the... Northlands... Tallin?")}
 
         [part]
-            story= _ "At Hidel’s call Tallin approached and knelt beside Hidel, bowing his head in respect."
+            story= _ "At Hidel’s call Tallin approached. He knelt beside, bowing his head in respect."
         [/part]
 
         {STORY_PART_TALLIN (_ "At your service, sir.")}
 
         {STORY_PART_HIDEL (_ "Eryssa has been... my charge ever... since she was born... I now return... to the earth... from which... I sprang... Please take care... of her.")}
 
-        {STORY_PART_TALLIN (_ "Rest assured, sir, I will place myself between Eryssa and any ill that strength or love may counter.")}
+        {STORY_PART_TALLIN (_ "I will.")}
 
         {STORY_PART_HIDEL (_ "Thank you... Tallin... Now I may rest... in... peace... May you... be victorious...")}
 
         [part]
-            story= _ "With these words Hidel closed his eyes and breathed his last. As Eryssa wept with Hidel’s head in her lap, Tallin approached her, hesitating, and then wrapped her gently in his arms."
+            story= _ "Then Hidel closed his eyes and took one last breath."
+        [/part]
+
+        {STORY_PART_ERYSSA (_ "He's dead!")}
+
+        [part]
+            story= _ "Slowly Tallin aproached her. He hesitated before wrapping her gently in his arms."
             music=love_theme.ogg
         [/part]
         [part]
-            story= _ "Sobbing against Tallin’s chest, Eryssa told him of the many ways Hidel had cherished and warded her since the hour of her birth. How he had comforted her childhood hurts; how in later years, he had guided her in the green paths of the forest, and taught her the deep mysteries of elven-kind."
+            story= _ "Sobbing against Tallin’s chest, Eryssa told him of the many ways Hidel had cherished and warded her and her sister since the hour of their birth. How he had comforted their childhood hurts; how in later years, he had guided them in the green paths of the forest, and taught them the deep mysteries of elven-kind."
```

Comment:
I think it's okay to have this line just be about her instead of her and Elenia.

Reply:
But I don't like how Elenia (who is her younger sister) disappears. Even if she's dead, I think the princess would mention her.

Reply:
Sure thing, you can go ahead and add more events/lines for Eryssa to talk about Elenia.

Reply:
You mean, leave this one or delete it and add some elsewhere? A lot of places you could probably skip it, but if it's something like their dad dying (Adopted dad) that'd make me think of my siblings.

Reply:
Somehow it feels a bit awkward to me here. You could revert this to the original and then add in extra snippets about how Hidel was so nice to her baby sister too?

Reply:
That's basically what I did...

Could you give me an example on what you mean?

Reply:
I guess it's mainly about the "hour of their birth" sounds like they're twins when they're not.

Reply:
Removing hour of birth.